docs/kubernetes/mailu/smtp.yaml

```diff
@@ -30,6 +30,9 @@ spec:
           - mountPath: /data
             name: maildata
             subPath: maildata
```

Comment:
`/data` is correctly removed in Dockerfile, since it is obsolete. I think it should be removed from here as well?

Reply:
Right, I've change it.